### DISCUSSION TITLE
mixpanel analytics code

### DISCUSSION THREAD

**Discussion by thomasyit** [squidfunk/mkdocs-material#5190] [Q&A] [answered]
Hi, i am trying to add the mixpanel javascript code but it doesnt seem to be working via netlify preview, as in the events are not being pick up in the mixpanel.  I follow the instructions over here https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#mkdocsyml and https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme .

**Comment by squidfunk** [comment, marked as answer]
Here's a PR adding support for Plausible Analytics, which is another third party provider. Maybe you can use the code included to debug where your implementation is not working.

- https://github.com/squidfunk/mkdocs-material/pull/4847

**Reply by thomasyit** [reply]
thanks, have resolve the issue.
